Dependencies.targets

```
@@ -12,41 +12,40 @@
     <PackageReference Update="Microsoft.EntityFrameworkCore.Relational" Version="$(EFCoreVersion)" />
     <PackageReference Update="Microsoft.EntityFrameworkCore" Version="$(EFCoreVersion)" />
 
-    <PackageReference Update="MySqlConnector" Version="2.1.2" />
+    <PackageReference Update="MySqlConnector" Version="2.2.0-rc.2" />
```

```suggestion
    <PackageReference Update="MySqlConnector" Version="2.2.0" />
```

2.2.0 (RTM) is now available: https://www.nuget.org/packages/MySqlConnector/2.2.0